pkg/kubelet/images/puller.go

```diff
@@ -58,8 +59,14 @@ func (pip *parallelImagePuller) pullImage(ctx context.Context, spec kubecontaine
 		}
 		startTime := time.Now()
 		imageRef, err := pip.imageService.PullImage(ctx, spec, pullSecrets, podSandboxConfig)
+		var size uint64
+		if err == nil && imageRef != "" {
+			// Getting the image size with best effort, ignoring the error.
+			size, _ = pip.imageService.GetImageSize(ctx, spec)
```

Comment:
what is the default timeout for this call?

Reply:
It uses --runtime-request-timeout duration, so default is 2min.

https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/